Makefile

```
@@ -1,25 +1,21 @@
-IMAGE_TAG=cloudbees/cjd-jcasc-demo:rolling
-INCLUDE_ALL_BUNDLED_PLUGINS?=true
+IMAGE_TAG=cloudbees/cjd-jcasc-demo
+DEVEL_SUFFIX=devel
+IMAGE_DEVEL_TAG=${IMAGE_TAG}:${DEVEL_SUFFIX}
 DOCKER_BUILD_FLAGS?=
 
 clean:
-	@if docker images cloudbees/cjd-jcasc-demo | awk '{ print $$2 }' | grep -q -F rolling; then docker image rm --force ${IMAGE_TAG} ; false; fi
+	@if docker images ${IMAGE_TAG} | awk '{ print $$2 }' | grep -q -F ${IMAGE_DEVEL_TAG}; then docker image rm --force ${IMAGE_DEVEL_TAG} ; false; fi
 
 build:
-	docker build -t ${IMAGE_TAG} ${DOCKER_BUILD_FLAGS} .
+	docker build -t ${IMAGE_DEVEL_TAG} ${DOCKER_BUILD_FLAGS} .
 
 run:
-	docker run -u root --rm -p 8080:8080 -p 50000:50000 \
-		-v /var/run/docker.sock:/var/run/docker.sock \
-		-e JAVA_OPTS="-Dcom.cloudbees.jenkins.cjp.installmanager.CJPPluginManager.allRequired=${INCLUDE_ALL_BUNDLED_PLUGINS}" \
-		${IMAGE_TAG}
+	docker run --rm -p 8080:8080 ${IMAGE_TAG}
 
 run-devel:
-	@if ls work ; then rm -rf work ; false; fi
+	rm -rf work
 	mkdir work
-	docker run -u root --rm -p 8080:8080 -p 50000:50000 \
-		-v /var/run/docker.sock:/var/run/docker.sock \
-		-e JAVA_OPTS="-Dcom.cloudbees.jenkins.cjp.installmanager.CJPPluginManager.allRequired=${INCLUDE_ALL_BUNDLED_PLUGINS}" \
+	docker run --rm -p 8080:8080 \
 		-e VERBOSE=true \
 		-v $(CURDIR)/work:/var/jenkins_home/ \
```

what about `/var/cloudbees-jenkins-distribution`?

---

Did not need it. It is a developer code.
Is it a symlink?

---

The Dockerfile is based on a CJD docker image whose JENKINS_HOME is `/var/cloudbees-jenkins-distribution` and not `/var/jenkins_home/` (`FROM cloudbees/cloudbees-jenkins-distribution:2.176.2.3`), so when you 
> It is a developer code.

Can I understand you are not interested in storing the JENKINS_HOME?

---

```suggestion
		-v $(CURDIR)/work:/var/cloudbees-jenkins-distribution/ \
```

Right, just forgot to patch it after updating the baseline